src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const KEYS = Object.values(robot);
+  const VALUES = Object.keys(robot);
+  const INVERSED_ROBOT = {};
+  const UNIQUE_KEYS = [ ...new Set(KEYS) ];
```

Comment:
The variable name 'UNIQUE_KEYS' is misleading. It actually contains the unique values of the 'robot' object. Consider renaming it to 'UNIQUE_VALUES' or something similar that accurately represents its contents.